tests/conftest.py

```diff
@@ -70,24 +70,24 @@ def settings():
 @pytest.fixture(scope="session")
 def api():
     # handle to make API calls
-    api = snappi.api(
-        location=utl.settings.location, ext=utl.settings.ext
-    )
+    api = snappi.api(location=utl.settings.location, ext=utl.settings.ext)
+    utl.configure_credentials(api, utl.settings.username, utl.settings.psd)
     yield api
     if getattr(api, "assistant", None) is not None:
         api.assistant.Session.remove()
 
 
-@pytest.fixture(scope="session")
-def cvg_api():
-    # handle to make Convergence API calls
-    api = snappi_convergence.api(
-        location=utl.settings.location, ext=utl.settings.ext
-    )
-
-    yield api
-    if getattr(api, "assistant", None) is not None:
-        api.assistant.Session.remove()
+# @pytest.fixture(scope="session")
```

Comment:
feel we can uncomment it as this will not hit

Reply:
Temporarily reduced coverage